Scenes/HUD/UnitStatusCard.gd

```diff
@@ -0,0 +1,43 @@
+extends ButtonStatusCard
+
+
+@export var _level_panel: ShortResourceStatusPanel
+
+
+func _ready():
+	SelectUnit.selected_unit_changed.connect(_on_selected_unit_changed)
+
+
+func _process(_delta):
```

Comment:
Do this in in SelectUnit.selected_unit_changed() signal.

Reply:
Done